learning/tour-of-beam/frontend/pubspec.yaml

```diff
@@ -31,15 +31,26 @@ dependencies:
   easy_localization_loader: ^1.0.0
   flutter:
     sdk: flutter
+  flutter_svg: ^1.0.3
   get_it: ^7.2.0
+  google_fonts: ^3.0.1
+  playground_components:
+    path: ../../../playground/frontend/playground_components
+  provider: ^6.0.3
+  shared_preferences: ^2.0.15
+  url_launcher: ^6.1.5
   url_strategy: ^0.2.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  integration_test:
+    sdk: flutter
```

Comment:
https://medium.com/flutter-senior/the-nicest-format-for-flutter-in-pubspec-yaml-d64349478dd9